core/common/src/main/java/alluxio/util/network/NettyUtils.java

```diff
@@ -66,9 +69,16 @@ public static EventLoopGroup createEventLoop(ChannelType type, int numThreads,
    * {@link ChannelType}.
    *
    * @param type Selector for which form of low-level IO we should use
-   * @return ServerSocketChannel matching the ChannelType
+   * @param isDomainSocket whether this is a domain socket server
+   * @return ServerSocketChannel matching the requirements
    */
-  public static Class<? extends ServerChannel> getServerChannelClass(ChannelType type) {
+  public static Class<? extends ServerChannel> getServerChannelClass(ChannelType type,
+      boolean isDomainSocket) {
```

Comment:
I have thought about it. The problem is that EPOLL and DOMAIN_SOCKET are not mutual exclusive. 

EPOLL and NIO are different ways to poll on sockets. but Domain socket is a different socket. It also sounds strange to put domain socket as something comparable with EPOLL and NIO.